libs/sdk-backend-bear/src/convertors/fromBackend/afm/result.ts

```diff
@@ -0,0 +1,47 @@
+// (C) 2020 GoodData Corporation
+import { DataValue } from "@gooddata/sdk-backend-spi";
+import { GdcExecution } from "@gooddata/api-model-bear";
+import { IPostProcessing } from "@gooddata/sdk-model";
+import { transformDateFormat } from "../../dateFormatting/dateFormatter";
+
+export type TransformedResult = {
+    readonly headerItems?: GdcExecution.IResultHeaderItem[][][];
+    readonly data: DataValue[][] | DataValue[];
+    readonly total: number[];
+    readonly offset: number[];
+    readonly count: number[];
+    readonly totals?: DataValue[][][];
+    readonly warnings?: GdcExecution.Warning[];
+};
+
+export function transformHeaderItems(
+    headerItems?: GdcExecution.IResultHeaderItem[][][],
+    postProcessing?: IPostProcessing,
+): GdcExecution.IResultHeaderItem[][][] | undefined {
+    if (!headerItems || !postProcessing) {
+        return headerItems;
+    }
+    return headerItems.map((headerItems1: GdcExecution.IResultHeaderItem[][]) => {
+        return headerItems1.map((headerItems2: GdcExecution.IResultHeaderItem[]) => {
+            return headerItems2.map((headerItem: GdcExecution.IResultHeaderItem) => {
+                if (postProcessing.dateFormat) {
+                    return transformDateFormat(headerItem, postProcessing.dateFormat);
+                }
+                return headerItem;
+            });
+        });
+    });
+}
+
+export function transformExecutionResult(
```

Comment:
This functionality should be in `sdk-backend` base. If the API is changed so that it is 

```ts
transformExecutionResult(dataView: IDataView, postProcessing?: IPostProcessing): IDataView;
```

Then it would be backend independent and thus moved to `sdk-backend-base` so that it can be used by multiple backends.

Reply:
I think moving a function like below to sdk-backend-base should be easier to be used in multiple backends, and I've moved such function to sdk-backend-base. Please have a look at it.

```
transformResultHeaders(
    resultHeaders: IResultHeader[][][],
    resultHeaderTransformer?: ResultHeaderTransformer,
    postProcessing?: IPostProcessing,
): IResultHeader[][][]
```